GraphStructure/Vertex.java

```diff
@@ -0,0 +1,16 @@
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Iterator;
+
+public class Vertex{
+	public Integer id;
+	public HashSet<Integer> adjvertex;
+	public Vertex(){
+		this.id=new Integer(-1);
+		this.adjvertex=new HashSet<Integer>();
+	}
+	public boolean ifAdj(Integer vid){
```

Comment:
Normally, we call it isAdjacent? Better use full name if it is not a common used naming.